src/main/kotlin/org/wa/rceditor/application/model/ProjectItem.kt

```diff
@@ -15,8 +14,6 @@ class ProjectItem(
         path: String,
         category: String) {
```

Comment:
Should this support lists? I know we only have examples of single categories, but technically there could be more according to the RC spec.